pygfx/materials/_mesh.py

```diff
@@ -108,6 +96,15 @@ def vertex_colors(self):
     def vertex_colors(self, value):
         self._store.vertex_colors = bool(value)
 
+    @property
+    def face_colors(self):
+        """Whether to use the vertex colors provided in the geometry."""
```

Comment:
"""Whether to use the face colors provided in the geometry."""

Reply:
Will include this fix.